package.json

```diff
@@ -1,14 +1,21 @@
 {
   "name": "@first-lego-league/ms-correlation",
-  "version": "0.0.4",
+  "version": "0.0.5",
   "description": "A library to help support MS with domains",
   "main": "index.js",
   "repository": "git@github.com:FirstLegoLeagueIL/ms-correlation.git",
   "author": "Roy Shmeuli <roysh9@gmail.com>",
-  "license": "MIT",
-  "private": false,
+  "license": "GPL-3.0",
+  "files": [
```

Comment:
npm/yarn would always include `package.json`, `README` and `LICENSE`. Also we can remove the yarn.lock we don't really use it on packages.